devicetypes/smartthings/aeon-multiwhite-bulb.src/aeon-multiwhite-bulb.groovy

```diff
@@ -30,6 +30,7 @@ metadata {
 
 		fingerprint mfr: "0371", prod: "0103", model: "0001", deviceJoinName: "Aeon LED Bulb 6 Multi-White" //US
 		fingerprint mfr: "0371", prod: "0003", model: "0001", deviceJoinName: "Aeon LED Bulb 6 Multi-White" //EU
+		fingerprint mfr: "0300", prod: "0003", model: "0004", deviceJoinName: "ilumin Tunalbe White"
```

Comment:
Is it "ilumi" or "ilumin"?

Also, typo in "Tunable"

Reply:
It is "ilumin". 

I will fix this Tunable.

Reply:
@greens  fixed